pkg/deployable/runner.go

```diff
@@ -19,6 +19,7 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
```

Comment:
Don't you need to include also the time for the divert and external resources deployment? Those are also happening as part of the runner

Reply:
On the initial spec it was only though to be on compose/commands/wait. Let's ping @jpf-okteto in order to check if we should also include those one